bundles/NewsletterBundle/src/Controller/Document/NewsletterController.php

```diff
@@ -197,6 +207,7 @@ public function getAvailableClassesAction(): JsonResponse
                 }
```

Comment:
Found out a bug related to `NewsletterActive` 
https://github.com/pimcore/pimcore/issues/14887

Depending on the outcome of that ticket, the changes should be reflected in this PR too

Reply:
This can be done after refactoring as we need time to decide if this is a demo fix or need refactoring on the core data type. please see https://github.com/pimcore/pimcore/issues/14887#issuecomment-1503020817